data/coalition/heliarch intro.txt

```diff
@@ -0,0 +1,1951 @@
+
+mission "Heliarch Containment 4-B"
+	name "Meet with Plortub"
+	description "Collect your payment for escorting the Heliarch transports."
+	landing
+	to offer
+		has "Heliarch Containment 4-A: done"
+		not "Lunarium: Questions: done"
+	source "Remote Blue"
+	destination "Station Cian"
+	on offer
+		conversation
+			`Hundreds of Heliarch agents pour out of the three Kimek transports, rushing in groups to separate parts of the city. They march in unison, weapons in hand. From the reactions of the locals, it's clear they were neither expecting such a thing, nor are they happy with the agents going about the city in that manner.`
+			`	When the last of the Heliarchs leave the transports, one of the Kimek captains inform you that you should now head to <destination> to meet with Consul Plortub collect your payment of <payment>, and to "avoid getting caught in any potential retaliation."`
+				accept
+	on complete
+		payment 495000
+		conversation
+			`You land in the station to find that <payment> have been transferred to you. Along with the credits comes a transmission from Plortup. "For my absence, forgive me you must, Captain. Hit Fourth Shadow again, criminals have, so organizing some ships from within the Ring of Power, I am."`
+			`	He goes on to explain that the attack was quite a large one, and much of the fleet stationed here was sent there to help contain it. What remained is helping with the <origin> operation, so the station is fairly devoid of a military presence at the moment. "A troubled week, this will be, but bother you with that, I will not. Very useful to us your support has been, Captain."`
+
+mission "Heliarch Containment 5"
+	name "Stop The Fleeing Ships"
+	description "Defeat the criminal ships that are attempting to flee. You must disable and board their ships in order to help the Heliarchs find the source of their weapons."
+	landing
+	to offer
+		has "Heliarch Containment 4-B: done"
+		not "Lunarium: Questions: done"
+	source "Station Cian"
+	on offer
+		"reputation: Lunarium" = -1000
+		event "lunarium fleeing rb"
+		conversation
+			`After the transmission with Plortup ends, the station is awfully quiet for a few hours. You head outside your ship to see if you can find something interesting to do, but you don't find anything particularly eye-catching, as the station holds few shops. You move to the job board, but find it completely empty. You also notice that the market's trade values are not updating, and any news system is non-functional.`
+			`	Suddenly, some of the few Heliarchs still stationed here rush past you heading for an office. They quickly reemerge and approach you.`
+			`	"Captain! Helped us, you have, yes? Aiding in the Remote Blue operation?" You nod. "Think you trustworthy, the consul does, and in dire need of a combat-ready ship, we are. Aid us, you must!"`
+			choice
+				`	"What's going on?"`
+				`	"My ship isn't exactly combat-ready to fight your own warships."`
```

Comment:
```suggestion
				`	"My ship isn't exactly combat-ready to fight your own warships."`
```
It feels a little odd to have you say this, but then also be forced to fight anyways. Maybe there should be some sort of decline option in this mission.

Reply:
Changed to `"What do you mean combat-ready? Are we being attacked?"`
I don't think this mission should be declineable at this point, it is the last mission of the chain and would frankly be inappropriate as I see it if you can simply dip on the Heliarchs here when you've been pretty willing. I don't thinki "forcing" at this point is really "forcing".